HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -2105,20 +2105,70 @@ def self.apply_hvac(runner, hpxml_bldg, weather, convert_shared_systems, unit_nu
       heat_pump.airflow_defect_ratio_isdefaulted = true
     end
 
-    # Fan power
+    # Fan model type
+    hpxml_bldg.heating_systems.each do |heating_system|
+      next unless heating_system.heating_system_type == HPXML::HVACTypeFurnace
+      next unless heating_system.fan_model_type.nil?
+      next if (not heating_system.distribution_system.nil?) && (heating_system.distribution_system.air_type == HPXML::AirTypeGravity)
+
+      if (not heating_system.attached_cooling_system.nil?) && (not heating_system.attached_cooling_system.compressor_type.nil?)
+        # Based on RESNET DX Modeling Appendix
+        heating_system.fan_model_type = (heating_system.attached_cooling_system.compressor_type == HPXML::HVACCompressorTypeSingleStage) ? HPXML::HVACFanModelTypePSC : HPXML::HVACFanModelTypeBPM
```

Comment:
For attached heating and cooling systems, grab the cooling compressor type first to default, otherwise follow the previous HEScore assumption (based on efficiency)